roles/science/tasks/mrtrix.yml

```diff
@@ -37,6 +16,35 @@
       - libtiff5-dev
       - libpng-dev
 
+
+- name: Create directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - /opt/quarantine/software/mrtrix3/3.0.4
+    - /opt/quarantine/software/mrtrix3tissue/5.2.9
```

Comment:
can you adjust this so its not moved from the top.

I have plans in the future to make ansible functions to do directory creation etc and I'd like to keep the plays as uuniform as possible